core/perf_test/CMakeLists.txt

```diff
@@ -113,3 +137,9 @@ IF(NOT Kokkos_ENABLE_OPENMPTARGET)
     CATEGORIES PERFORMANCE
   )
 ENDIF()
+
+KOKKOS_ADD_EXECUTABLE_AND_TEST(
```

Comment:
FIXME: Do not use `KOKKOS_ADD_EXECUTABLE_AND_TEST` (or any Trilinos specific bits).
Implement this in pure CMake and protect with `IF (NOT KOKKOS_HAS_TRILINOS)`